src/main/java/com/gtnewhorizons/modularui/api/fluids/IFluidTankLong.java

```diff
@@ -0,0 +1,53 @@
+package com.gtnewhorizons.modularui.api.fluids;
+
+import static com.google.common.primitives.Ints.saturatedCast;
+
+import net.minecraftforge.fluids.Fluid;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.FluidTankInfo;
+import net.minecraftforge.fluids.IFluidTank;
+
+public interface IFluidTankLong extends IFluidTank {
+
+    long fill(Fluid fluid, long amount, boolean simulate);
+
+    FluidStack drain(long amount, boolean simulate);
```

Comment:
Now `IFluidTankLong#drain(long, boolean)` and `IFluidTank#drain(int, boolean)` look very similar but have inverted boolean. How about inverting boolean `simulate` to match to existing `IFluidTank`?

Reply:
fixed